web_development_101/installations/prerequisites.md

```diff
@@ -128,18 +128,23 @@ For step-by-step instructions, please follow this [installation guide](https://t
 </details>
 
 <details markdown="block">
-<summary class="dropDown-header">Windows 10 WSL (Not Recommended)
+<summary class="dropDown-header">Windows 10 WSL (Not Supported and Not Recommended)
 </summary>
   
-**Please note**: *Windows Subsystem for Linux is **highly discouraged** for those unfamiliar with Linux. Specifically, those unfamiliar with with the Linux Command Line. Please consider installing Linux in a virtual machine or dual-boot*.
+**Please note**: *Windows Subsystem for Linux is **highly discouraged and not recommended** for those unfamiliar with Linux. Specifically, those unfamiliar with with the Command Line. Please consider installing Linux in a virtual machine or dual-boot*.
```

Comment:
I think we want some verbiage about advanced users of Windows too. Cause that has been one of the main sources of problems with it.

Reply:
@ChadKreutzer Let me know if the edit in the following commit is what you were going for.